### PR TITLE
Adding Python quickstart.

### PR DESCRIPTION
**Pull request by rdegges**
Adding the Python quickstart section with code samples. Ready to go afaik.

### CODE REVIEW

File: source/quickstart.rst [hunk 1]
```diff
@@ -12,7 +12,11 @@
 
     This quickstart will get you up and running with Stormpath in about 7 minutes and give you a good initial feel for the Stormpath .NET SDK.
 
-.. only:: not (csharp or vbnet)
+.. only:: python
+
+    This quickstart will get you up and running with Stormpath in about 7 minutes and give you a good initial feel for the Stormpath Python SDK.
+
+.. only:: not (csharp or vbnet or python)
```
Comment (nbarbettini) [inline source/quickstart.rst]: Probably should be `only:: curl` at some point?

Comment (rdegges) [inline source/quickstart.rst]: I suppose so -- should we try to extract that out or wait until everyone gets theirs in? oO

**Comment by nbarbettini** [comment]
Woohoo :tada:

File: source/quickstart.rst (hunk 1 above)
Comment (nbarbettini) [inline source/quickstart.rst]: Long day, I meant `only:: rest`. I don't think there's any harm in changing it now. @jakubswiatczak thoughts?

Comment (rdegges) [inline source/quickstart.rst]: I just modified it to adhere to this. Good call! Looks a lot cleaner.

**Comment by rdegges** [comment]
Ping @nbarbettini @jakubswiatczak for review.

**Pull request merged**